docs/snippets/defaults_generic_ad_platforms.mdx

```diff
@@ -0,0 +1,14 @@
+```
+generic_creative_ad_server:
+  emissions_per_creative_request_per_geo_gco2pm:
```

Comment:
@bokelley Do we also want to move `default_emissions_per_creative_request_gco2pm` constant into this file from `docs/snippets/defaults_ad_platform.mdx` or combine these files?